### ISSUE TITLE
[Help Wanted] Unable to create an account and sign in to self-hosted server using Web app not self hosted (Failed to fetch)

### ISSUE THREAD

**Issue by sebgeekinside** [standardnotes/forum#3833]
When I try to create an account and sign in to my self-hosted server using Web app (https://app.standardnotes.com), I've got the message: Failed to fetch.

Same message using the Android app.

I use Traefik v2 as reverse-proxy and as certificates resolver in order to secure HTTP traffic of my server.
The URL https://my.domain.com/healthcheck returns OK.
SSL checker of the URL https://my.domain.com/ returns OK.

Is somebody can help me to resolve the problem?
Maybe a Traefik configuration to make (middleware ?)...

**Comment by piit79** [comment]
Do you see any cookie errors in the Developer tools Console by any chance? If you do, please refer to this issue: https://github.com/standardnotes/server/issues/1068

tl;dr: You need to set `COOKIE_DOMAIN=<your-API-hostname>` in the `.env` file or in `server.environment` in the `docker-compose.yml` file.
